stockfish/models.py

```diff
@@ -31,6 +31,7 @@ def __init__(
             "UCI_Chess960": "false",
             "UCI_LimitStrength": "false",
             "UCI_Elo": 1350,
+            "UCI_ShowWDL": "true",
```

Comment:
Let's leave default `false`

Reply:
Let's create methods to set this property: `enable_show_wdl` and `disable_show_wdl`